app/src/main/kotlin/cz/covid19cz/erouska/ui/exposure/ExposuresFragment.kt

```diff
@@ -31,7 +31,7 @@ class ExposuresFragment : BaseFragment<FragmentExposuresBinding, ExposuresVM>(
         symptoms_content.text = AppConfig.riskyEncountersWithSymptoms
         no_symptoms_content.text = AppConfig.riskyEncountersWithoutSymptoms
 
-        viewModel.checkExposures(args.demo)
+        viewModel.zcheckExposures(args.demo)
```

Comment:
typo?

Reply:
Jo, commitnul jsem na to fix, ale zapomněl pushnout, mělo by to být už dobrý